app/src/static/online-status.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<style>
+  .center {
+    margin: auto;
+    text-align: center;
+    width: 50%;
+  }
+
+  .hide {
+    display: none;
+  }
+</style>
+
+<body>
+  <div id="checkingInternet" class="center">
+    <h1>Checking Internet Connectiong...</h1>
+  </div>
+  <div id="online" class="center hide">
+    <h1>You are now connected to Internet!</h1>
+  </div>
+  <div id="noInternet" class="center hide">
+    <img src="nointernet.gif" width="110" height="90">
```

Comment:
Let's avoid the binary GIF; it's heavy and could be licensed.

What about using an emoji?

- ❌
- 📶
- ⚠